<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TelaCardapio</class>
 <widget class="QWidget" name="TelaCardapio">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>635</width>
    <height>642</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
	background-color: rgb(249, 245, 245);
	font: 12pt &quot;Segoe UI&quot;;
}

QLabel {
	font-size: 14pt;
	color:  black;
padding: 5px;
}

QPushButton {
	background-color: rgb(178, 34, 34);
	color: white;
	border: none;
	border-radius: 8px;
	padding: 12px 25px;
	font-weight: bold;
font-size: 14pt;
}

QPushButton:hover {
	background-color: rgb(204, 51, 51);
}

QPushButton:pressed {
	background-color: rgb(153, 31, 31);
}

QTableWidget {
background-color: white;
	color: #333;
gridline-color: #eee;
border: 1px solid #ccc;
alternate-background-color: rgb(247, 247, 247);
selection-background-color: rgb(255, 229, 229);
selection-color: black;
border-radius: 6px;
}

QHeaderView::section {
background-color: rgb(178, 34, 34);
color: white;
font-weight: bold;
font-size: 12pt;
padding: 6px;
border: none;
}

QScrollBar:vertical {
border: none;
background: #f0f0f0;
width: 10px;
margin: 0px;
}

QScrollBar::handle:vertical {
background: #ccc;
min-height: 20px;
border-radius: 5px;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="labelBemVindo">
     <property name="text">
      <string>Bem-vindo!</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <property name="spacing">
      <number>20</number>
     </property>
     <item row="7" column="2" colspan="2">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="botaoAdicionar">
         <property name="text">
          <string>Adicionar produto</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="botaoFinalizar">
         <property name="text">
          <string>Finalizar pedido</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label">
       <property name="styleSheet">
        <string notr="true">font-size: 24px;
font-weight: bold;
color: rgb(183, 28, 28);
letter-spacing: 2px;</string>
       </property>
       <property name="text">
        <string>APERITIVOS</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="3">
      <widget class="QTableWidget" name="tabelaBebidas">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>400</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>400</height>
        </size>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SelectionMode::SingleSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
       </property>
       <property name="columnCount">
        <number>2</number>
       </property>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <column/>
       <column/>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="label_2">
       <property name="styleSheet">
        <string notr="true">font-size: 24px;
font-weight: bold;
color: rgb(183, 28, 28);
letter-spacing: 2px;</string>
       </property>
       <property name="text">
        <string>BEBIDAS</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="2">
      <widget class="QTableWidget" name="tabelaAperitivos">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>400</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>400</height>
        </size>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SelectionMode::SingleSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
       </property>
       <property name="columnCount">
        <number>2</number>
       </property>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <column/>
       <column/>
      </widget>
     </item>
     <item row="6" column="2">
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
